src/test/java/com/dungeoncrawler/Javiarenka/HeroTest.java

```diff
@@ -60,16 +59,4 @@ void testAttackAndNotKillMonster() {
         testedHero.attack(testMonster);
         assert (testMonster.isAlive());
     }
-
-    @Test
-    void testGetMoneyFromEmptyPocket(){
-        testedHero.setMoney(0);
-        Exception exception = assertThrows(NoMoreMoneyException.class, () -> testedHero.removeMoney(20));
-
-        String expectedMessage = NoMoreMoneyException.MESSAGE;
-        String actualMessage = exception.getMessage();
-
-        assertTrue(actualMessage.contains(expectedMessage));
-    }
```

Comment:
Dobrze by było zastąpić ten test podobnym, tylko w nowej klasie

Reply:
To prawda. Niestety ja nie potrafię w TDD, wiec jak już będzie cos działać, to zastanowię się nad testami (jak mus to mus).